library/ssl_tls13_client.c

```diff
@@ -939,7 +939,7 @@ int mbedtls_ssl_write_pre_shared_key_ext( mbedtls_ssl_context *ssl,
             }
 
             obfuscated_ticket_age =
-                (uint32_t)( now - ssl->session_negotiate->ticket_received ) +
+                (uint32_t)( ( now - ssl->session_negotiate->ticket_received ) * 1000 ) +
```

Comment:
It seems a fix is needed at line 934 as well as `now - ssl->session_negotiate->ticket_received` (age in seconds) seems to be compared to the number of milliseconds in one week.

Reply:
Agreed. The intention is to keep the fix simple. 

Regarding the L934, the fix is not straightforward. If we just return `MBEDTLS_ERR_SSL_SESSION_TICKET_EXPIRED`, the request will fail. But a better approach is to fall back to full handshake (as if there is no session ticket). What do you think on this approach?

Reply:
Falling back to the full handshake is certainly better than having the exchange fail

Reply:
I agree with Ron that this change needs to be aligned with the calculation in line 934